### PR TITLE
feature/NM-18

### PR DESCRIPTION
**Pull request by amolcan**
Changed format of the displayed data.

### CODE REVIEW

File: controllers/comments.controller.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ function getCommentsCountForUserName(req, res, next) {
 
     comments
         .then(data => {
-            res.json(data);
+            let commentsCount = data;
+            res.json(JSON.parse(`{"commentsCount":${commentsCount}, "userName":"${userName}", "startDate":"${startDate}", "endDate":"${endDate}"}`));
```
Comment (ajancus) [inline controllers/comments.controller.js]: Didnt we agreed on that controller will have clean methods and all logic (even creating json response) will be in service?

Comment (amolcan) [reply controllers/comments.controller.js]: Yes, sorry I forgot.

**Comment by amolcan** [comment]
Because there is no endDate in REST call in GitHub API, there is only since parameter, which can be used as a startDate.
https://developer.github.com/v3/pulls/comments/#list-comments-in-a-repository

File: services/comments.service.js [hunk 2]
```diff
@@ -16,16 +16,17 @@ function getCommentsCountForUserName(userName, startDate, endDate) {
...
+        return getCommentsSum(repositoriesCommentsArr, userName, startDate, endDate);
     }
 }
 
-function getCommentsSum(repositoriesCommentsArr) {
+function getCommentsSum(repositoriesCommentsArr, userName, startDate, endDate) {
     return Promise.all(repositoriesCommentsArr)
-    .then(data => data.reduce((a, b) => a + b));
+    .then(data => data.reduce((a, b) => a + b))
+    .then(commentsCount => JSON.parse(`{"commentsCount":${commentsCount}, "userName":"${userName}", "startDate":"${startDate}", "endDate":"${endDate}"}`));
```
Comment (tjalc) [inline services/comments.service.js]: I would prefer to have userName as first value. It's more readable

Comment (amolcan) [reply services/comments.service.js]: Done.

**Pull request merged**